### DISCUSSION TITLE
Is the project in active development?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [PotionApps/potionx#5] [General]
This seems like a super useful project and I was wondering whether or not it will keep getting updates? Most notably the planned `Restricting access to routes on the frontend` and `Pulumi recipe for other providers`, and building upon the project?

I'd love to keep using it and maybe even contributing but that seems outside of my current abilities

**Comment by vince-roy** [comment]
Glad to hear you find it useful! I'm happy to keep updating it if there is interest as I use it all the time anyway.

Aside from the issues that I will sort out shortly, what other features are of interest?

**Reply by Nikola-Milovic** [reply]
I am still playing around with it, and it has pretty much everything I'd need to get a project running asap. Keeping it up to date with your personal usage would probably be enough as far as I am concerned.  

Maybe tailwind 3.0 could be a nice addition. 

I am just a bit confused about the generated Admin for the frontend. Is that a sample project, as its quite large? I would expect the name to match my answer during init, like if my app is `example` and I get `example-graphql` and `example-web`, the Vue project would be `example-frontend` .

**Reply by vince-roy** [reply]
I'm using Tailwind 3.0 in my current projects so that's an easy one and makes sense.

The generated admin frontend was an attempt to get a default admin area out of the box (similar to Django admin or https://redwoodjs.com/) for user management and other generic CRUD. 

For most projects I have admin and then public or marketing folders that hold public-facing applications. For example, admin.example.com would show the admin folder, www.example.com would show the marketing folder and app.example.com would show the public folder. 

I suppose I could rename the frontend to {project-name}-frontend. Having it standardized as frontend simplified some of the deployment functions.

**Reply by Nikola-Milovic** [reply]
That's great!

So you create other projects as you see fit under the frontend directory? 

frontend
|- admin
|- public facing project
|- marketing project/ landing page

What kind of additional setup does this require afterward? As I saw `frontend/admin` being mentioned in several places in the lib, `mix.exs`

Sorry for all the inquiries, still getting a hang of the project and the ecosystem, also thanks for sharing the project!

**Reply by vince-roy** [reply]
Yes exactly! I use this structure over at pagespace.app.

/frontend/public is what you see over at pagespace.app (with SSR) and then /frontend/admin is used for another subdomain to manage analytics and users.

It just requires a couple of tweaks to Earthfile and the Phoenix router. Nothing in mix.exs. I'll add some documentation for this.

Thanks for the inquiries and feedback, this is how projects get better!

**Reply by Nikola-Milovic** [reply]
Would love to read more about this approach. Please do keep me updated

**Reply by vince-roy** [reply]
Sounds good. Should have some updates and fixes up this weekend.

**Reply by Nikola-Milovic** [reply]
After playing around with the project for a bit longer, is it possible to detach UI from the phoenix? For example have  NuxtJS/Vue for the frontend and just share the Graphql models. I experimented a bit but couldn't manage to get contents of `shared` into my project when running. If I manage to get this to work, would everything else work as expected? As session plug directly calls redirects from what I've seen

**Reply by vince-roy** [reply]
You can definitely detach from Phoenix and use the Elixir side just as a GraphQL API. When you run "npm run dev" it starts the usual Vite processes accessible at localhost:3000 that should work or only require minor modifications. They only communicate via GraphQL anyway.
